package.json

```diff
@@ -38,6 +38,8 @@
     "@types/react-router-dom": "^4.2.7",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
+    "react-docgen-typescript": "^1.6.2",
+    "react-styleguidist": "^7.1.0",
```

Comment:
I'd recommend removing the `^` at the front of these two dependencies to freeze it on specific versions.

Reply:
Why these packages and not others?